### ISSUE TITLE
Can someone help me with installation

### ISSUE THREAD

**Issue by Athirakr94** [facebookresearch/sound-spaces#110]
I have followed the instructions (https://github.com/facebookresearch/sound-spaces/blob/main/INSTALLATION.md ) but to confirm we need to install habitatlab, habitat sim and clone sound spaces to same root folder. 
To download dataset I saw command as python -m habitat_sim.utils.datasets_download --uids habitat_test_scenes --data-path data/ But in habitatsim there is no such script seen. Is there any more step we need to do

**Comment by ChanganVR** [comment]
Yes. You will need to install these repos first. If it does not work, please post an issue in the Habitat repo.

**Comment by Athirakr94** [comment]
Any installation procedure exist that captures all details or any resources to help set this up
